workspaces/ui-v2/src/pages/onboarding/Onboarding.tsx

```diff
@@ -0,0 +1,202 @@
+} from '<src>/hooks/useCapturesHook';
+
+const useStyles = makeStyles((theme) => ({
+  body: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    width: '60%',
+    marginTop: theme.spacing(2),
+  },
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+interface OnboardingData {
+  specs_shared_with_team: number;
+  specs_shared_not_with_team: number;
+  specs_uploaded_by_gitbot: number;
+  coverages_uploaded: number;
+}
+
+export default function Onboarding() {
+  const classes = useStyles();
+
+  const specId = useAppSelector(
+    (state) => state.metadata.data?.specificationId!
+  );
+
+  const captures = useCaptures();
+  const capturesService = React.useContext(CapturesServiceContext);
+
+  const captureStatuses = useAsyncMemo(async () => {
+    if (!captures.loading && !captures.error && capturesService) {
+      const captureStatuses = await Promise.all(
+        captures.captures.map((cap) =>
+          capturesService.getCaptureStatus(cap.captureId)
+        )
+      );
+
+      debugger;
+
+      return captureStatuses.reduce(
+        (accum, curr) => {
+          if (curr.metadata?.taskName === 'test') {
+            accum.test += curr.interactionsCount;
+          } else {
+            accum.regular += curr.interactionsCount;
+          }
+
+          return accum;
+        },
+        { regular: 0, test: 0 }
+      );
+    } else {
+      return { regular: 0, test: 0 };
+    }
+  }, [captures, capturesService]);
+
+  const {
+    backendApi: { domain: baseDomain },
+  } = useAppConfig();
+
+  const onboardingData = useAsyncMemo(async () => {
+    if (baseDomain && specId) {
+      const onboardingResponse = await fetch(
+        `${baseDomain}/api/public-specs/onboarding-by-analytics-id/${specId}`
+      );
+      const data: {
+        by_person: { [key: string]: OnboardingData };
+      } = await onboardingResponse.json();
+      return Object.entries(data.by_person).reduce(
+        (accum, [person_id, data]) => {
+          Object.entries(data).forEach(([k, v]) => {
+            if (v > 0) {
+              accum[k as keyof typeof accum].total += v;
+              accum[k as keyof typeof accum].users += 1;
+            }
+          });
+          return accum;
+        },
+        {
+          specs_shared_with_team: {
+            total: 0,
+            users: 0,
+          },
+          specs_shared_not_with_team: {
+            total: 0,
+            users: 0,
+          },
+          specs_uploaded_by_gitbot: {
+            total: 0,
+            users: 0,
+          },
+          coverages_uploaded: {
+            total: 0,
+            users: 0,
+          },
+        }
+      );
+    }
+  }, [baseDomain, specId]);
+
+  return (
+    <Page>
+      <Page.Navbar />
+      <Page.Body>
+        <div className={classes.root}>
+          {onboardingData ? (
+            <div className={classes.body}>
+              <Typography variant="h3" align="center">
```

Comment:
In general, I think we've been trying to move away from the typography component. Some notes around html / css structure:
- this is a list, we should use ul/li elements (with a [style reset](https://github.com/opticdev/optic/blob/develop/workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx#L162-L166)) - browsers, screen readers can intepret ul / li elements as a list for better accessibility / usability
- we could do a better job of having more sane defaults for p tags / typography components. Usually what I do for "consistency" is to copy paste similar stylings. The changelog history page is a page we've recently done which encompases a lot of what we want to do with styling in the FE https://github.com/opticdev/optic/blob/develop/workspaces/ui-v2/src/pages/changelogHistory/ChangelogHistory.tsx